### PR TITLE
lame-windows.0.3.3

### PR DESCRIPTION
**Pull request by toots**
I'm thinking of adding the modules provided by savonet. All our configuration scripts follow the usual `./configure` pattern for cross-compiling, i.e.:
* Pass `--host=<host triplet>`
* Expect compiler tools to be prefixed by `<host triplet>-`

Thus, I had to change the following:
* Export host triplet in `conf-gcc-windows`
* Link compiler binaries as `${host}-${binary}` in `PATH`-available `bin/` directory

Overall, this looks pretty clean to me, hopefully y'all can take it!

### CODE REVIEW

File: packages/lame-windows.0.3.3/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-lame"
+build: [
+  ["./configure" "--host=%{conf-gcc-windows:host}%" "--prefix=%{prefix}%" "OCAMLFIND_TOOLCHAIN=windows"]
+  [make]
+]
+install: [
+  ["env" "OCAMLFIND_TOOLCHAIN=windows"
+   "make" "install"]
```
Comment (whitequark) [inline packages/lame-windows.0.3.3/opam]: Are you completely certain this doesn't put any files in `%{prefix}%` outside of `%{prefix}%/windows-sysroot`? Can you verify this by doing a `make install DESTDIR=...`?

Comment (toots) [reply packages/lame-windows.0.3.3/opam]: Yeah, `install` trickles down to `libinstall`, which is pure `ocamlfind`:
```
libinstall:     all
        $(QUIET)printf "\nInstalling library with ocamlfind\n"
        $(OCAMLFIND) install $(OCAMLFIND_INSTFLAGS) $(RESULT) META $(LIBINSTALL_FILES)
        $(QUIET)printf "\nInstallation successful.\n"
```

Comment (toots) [reply packages/lame-windows.0.3.3/opam]:
```
[lame-windows: env make] Command started
+ env "OCAMLFIND_TOOLCHAIN=windows" "make" "install" (CWD=/Volumes/Home/toots/.opam/4.04.0+32bit/build/lame-windows.0.3.3)
- /Library/Developer/CommandLineTools/usr/bin/make -C src install
- make[2]: `liblame_stubs.a' is up to date.
- i686-w64-mingw32.static-ocamlmklib      -L/Volumes/Home/toots/.opam/4.04.0+32bit/lib    \
- 				-o lame_stubs  lame_stubs.o -lmp3lame -lm \
-
- i686-w64-mingw32.static-ocamlc -a -dllib dlllame_stubs.so       -ccopt -L/Volumes/Home/toots/.opam/4.04.0+32bit/lib  -ccopt -L/Volumes/Home/toots/.opam/4.04.0+32bit/lib     \
- 				-cclib -llame_stubs -cclib -lmp3lame -cclib -lm  -o lame.cma  lame.cmo
- make[2]: `liblame_stubs.a' is up to date.
- make[2]: `lame.cmxa' is up to date.
- i686-w64-mingw32.static-ocamlc -a -g -w +A@5-7@8@11@12@20-35-44-45 lame_dynlink.mli lame_loader.ml -o lame_loader.cma
- rm -f lame_dynlink.cmi
-
- Installing library with ocamlfind
- ocamlfind install  lame META lame.mli lame_dynlink.mli lame.cmi lame_loader.cmi lame.cma lame_loader.cma lame.cmxa lame.cmx lame_loader.cmx lame.a liblame_stubs.a lame.cmxs lame_loader.cmxs
-
- Installation successful.
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame_loader.cmxs
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame.cmxs
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/liblame_stubs.a
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame.a
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame_loader.cmx
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame.cmx
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame.cmxa
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame_loader.cma
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame.cma
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame_loader.cmi
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame.cmi
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame_dynlink.mli
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/lame.mli
- Installed /Volumes/Home/toots/.opam/4.04.0+32bit/windows-sysroot/lib/lame/META
∗  installed lame-windows.0.3.3
```

**Comment by whitequark** [comment]
Thanks!

**Pull request merged**